Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.h

```diff
@@ -542,6 +542,9 @@ class NetworkConnectionToWebProcess final
 #endif
 
     HashMap<WebTransportSessionIdentifier, Ref<NetworkTransportSession>> m_networkTransportSessions;
+#if ENABLE(LAUNCHSERVICES_SANDBOX_EXTENSION_BLOCKING)
+    std::optional<String> m_pendingDisplayName;
```

Comment:
Case in point, it holds a String so it can only be destroyed on the thread where this string was allocated.

Reply:
Also, we usually do not use std::optional with `String` since it is already nullable.